src/openassetio-python/CMakeLists.txt

```diff
@@ -112,15 +113,19 @@ if (OPENASSETIO_ENABLE_TESTS AND Python_Interpreter_FOUND)
         OUTPUT
         "${venv_python_dir}/pytest"
         COMMAND
-        "${installed_pip}" install -r "${PROJECT_SOURCE_DIR}/tests/requirements.txt"
+        "${installed_pip}" download --destination-directory ${download_dir} -r "${PROJECT_SOURCE_DIR}/tests/requirements.txt"
```

Comment:
[fix] Can we add some notes on why we're downloading these? (ie, as the wheel build insists on installing things).

Reply:
Will add a note as I refactor the dependencies as mentioned below.

Reply:
Addressed in https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/309/commits/d5e0db78ef4e78635c7d3343eb825a936d2ccaa6